airbyte-cdk/python/airbyte_cdk/sources/concurrent_source/concurrent_source_adapter.py

```diff
@@ -58,3 +61,45 @@ def _select_abstract_streams(self, config: Mapping[str, Any], configured_catalog
             if isinstance(stream_instance, AbstractStreamFacade):
                 abstract_streams.append(stream_instance.get_underlying_stream())
         return abstract_streams
+
+    def _convert_to_concurrent_stream(
```

Comment:
@lazebnyi can you add some tests to covert this new logic?

Reply:
added